java/code/src/com/redhat/rhn/common/db/datasource/xml/System_queries.xml

```diff
@@ -2419,6 +2419,17 @@ ORDER BY UPPER(COALESCE(X.SERVER_NAME, '(none)')), X.ID
   <elaborator name="entitlements" />
 </mode>
 
+<mode name="target_systems_for_maintenance_schedule" class="com.redhat.rhn.frontend.dto.EssentialServerDto">
+    <query params="user_id, schedule_id">
+        SELECT s.id, s.name
+        FROM rhnServer s JOIN rhnUserServerPerms usp on s.id = usp.server_id
+        WHERE usp.user_id = :user_id
+            AND (s.maintenance_schedule_id != :schedule_id
```

Comment:
Maybe better to check first if it is `null` and then by comparing by `schedule_id` just for the sake of performance?